### DISCUSSION TITLE
FormGroup cannot get value of TextField if it's focus

### DISCUSSION THREAD

**Discussion by minhson95th** [NativeScript/NativeScript#9656] [Q&A]
I want get value of TextField by FormGroup , but if i type value in TextField, and click button, FormGroup can not get value of current TextField, when i change to focus next TextField, now FormGroup can get value. How can i get value of TextField without change focus?
![Capture](https://user-images.githubusercontent.com/46039866/138386604-8accf963-bff9-426b-9b56-023629ce5d76.JPG)

![Capture1](https://user-images.githubusercontent.com/46039866/138386611-7064923a-a0bd-49c5-a19a-f7b9312ef2b6.JPG)

**Comment by rigor789** [comment]
You probably want to change the `commitMode` to use `Immediate` instead of `OnLostFocus`

**Comment by minhson95th** [comment]
> You probably want to change the `commitMode` to use `Immediate` instead of `OnLostFocus`

I use Nativescript with Angular, how can I do that?

**Comment by rigor789** [comment]
Please post your code you've used - I thought you were using RadDataForm (you still may be?), but without seeing the code I'm not sure.

If so, you would just add `commitMode="Immediate"` on the element.
